<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockFcdCtl</class>
 <widget class="QDockWidget" name="DockFcdCtl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>179</width>
    <height>237</height>
   </rect>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>FCD Controls</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lnbLabel">
        <property name="text">
         <string>LO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="lnbSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Local oscillator of up or down converter</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> MHz</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-2000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>7900.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lnaLabel">
        <property name="text">
         <string>LNA gain</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="lnaComboBox">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>+49.0 dB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+30.0 dB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+25.0 dB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+20.0 dB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+17.5 dB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+15.0 dB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+12.5 dB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+10.0 dB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+7.5 dB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+5.0 dB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+2.5 dB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+0.0 dB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-1.0 dB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-2.5 dB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-5.0 dB</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="corrGroupBox">
      <property name="title">
       <string>Corrections</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="freqCorrLabel">
         <property name="text">
          <string>Freq</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="freqCorrSpinBox">
         <property name="suffix">
          <string> ppm</string>
         </property>
         <property name="minimum">
          <number>-200</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>-115</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="iqGainLabel">
         <property name="text">
          <string>Gain</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="iqGainSpinBox">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.000030000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="iqPhaseLabel">
         <property name="text">
          <string>Phase</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="iqPhaseSpinBox">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.000030000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
